### PR TITLE
Screen4

### PR DESCRIPTION
**Pull request by sau9458**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,27 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import {Switch,BrowserRouter,Route,Link} from 'react-router-dom';
+import {browserHistory  } from 'react-dom';
+import {Switch,HashRouter,Route,Link} from 'react-router-dom';
 import WrappedNormalLoginForm from './component/Login.js';
 import error from './component/error.js';
 import WrappedRegisterForm from './component/Register.js';
```
Comment (arun-kumar-codes) [inline src/App.js]: no need to add .js

File: src/HomePage.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+.ant-tabs-nav.ant-tabs-nav-animated{
+    width:75%;
+}
+
+.ant-tabs-tab{
+width:22%;
+text-align: center;
+color:black;
+}
+
```
Comment (arun-kumar-codes) [inline src/HomePage.css]: remove unnecessary  spcaces

File: src/App.js (hunk 1 above)
Comment (sau9458) [reply src/App.js]: ok sir

File: src/HomePage.css (hunk 2 above)
Comment (sau9458) [reply src/HomePage.css]: ok sir

File: src/component/Calendar.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+<Col id="col-calendar" span={10} offset={6}>
+  <div id="content">
+      <div id="text-calendar">
+      On 8 November 2016, the Government of India announced the demonetisation
+       of all ₹500 (US$7.80) and ₹1,000 (US$16) banknotes of the Mahatma Gandhi
+        
+      </div>
+        <h1 id="feb">February</h1>
+         <Collapse bordered={false} >
+               <Panel header="13"  key="1">
```
Comment (arun-kumar-codes) [inline src/component/Calendar.js]: create this using lodash loop

File: src/component/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    <span >Profile</span></span>}key="4"></TabPane>
+  </Tabs>
+
+  
+ 
+    );
+  }
+}
+
+const WrappedHomePage = Form.create()(HomePage);
```
Comment (arun-kumar-codes) [inline src/component/HomePage.js]: why Form.create is used????
i can't see any form here

File: src/component/HomePage.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+export default WrappedHomePage;
```
Comment (arun-kumar-codes) [inline src/component/HomePage.js]: there is no need WrappedHomePage  
this can be HomePage

File: src/component/HomePage.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Col,Form, Icon, Input, Button,Tabs} from 'antd';
+import WrappedCalandar from './Calendar'
+
+const TabPane = Tabs.TabPane;
+class HomePage extends React.Component {
+  render(){
+    return(
+      
+<Tabs style={{width:'100%'}} >
```
Comment (arun-kumar-codes) [inline src/component/HomePage.js]: format your code

**Pull request merged**